src/CssInliner.php

```diff
@@ -713,6 +713,8 @@ private function getCssSelectorPrecedence(string $selector): int
      *
      * @param string $css CSS with comments removed
      *
+     * @psalm-return array<array-key, array<string, string>> Array of string sub-arrays with the keys
```

Comment:
I think there should be no description here - the description belongs with `@return` - if the description were to change, these could be liable to become out-of-step.

Reply:
Oh, you're right. I'll fix it.